libbeat/processors/actions/decode_json_fields_test.go

```diff
@@ -0,0 +1,100 @@
+package actions
+
+import (
+	"testing"
+
+	"github.com/elastic/beats/libbeat/common"
+	"github.com/elastic/beats/libbeat/logp"
+	"github.com/stretchr/testify/assert"
+)
+
+var fields = [1]string{"msg"}
+var config, _ = common.NewConfigFrom(map[string]interface{}{
+	"fields": fields,
+})
+
+func TestMissingKey(t *testing.T) {
+	input := common.MapStr{
+		"pipeline": "us1",
+	}
+
+	actual := getActualValue(t, config, input)
```

Comment:
These test cases that expect an error should also assert that an error occurred. You can use [`assert.Error()`](https://godoc.org/github.com/stretchr/testify/assert#Error).